db/dcrpg/queries.go

```diff
@@ -59,6 +59,11 @@ func closeRows(rows *sql.Rows) {
 	}
 }
 
+// storedAgendas holds the current state of agenda data already saved in the
+// db. This helps track changes in the lockedIn and activated heights when they
+// happen.
+var storedAgendas map[string]dbtypes.MileStone
```

Comment:
`storedAgendas` does not seem to have any purpose.

Also, queries.go is not the best spot for a package-level variable, and `InsertVotes` should probably not have the responsibility of managing this map.